frontend/src/routers/AppRouter/AppRouter.tsx

```diff
@@ -72,10 +75,11 @@ export const AppRouter = () => {
 		useState<GetWorkspaceDropdownOptionsQuery>()
 	const [projectListData, setProjectListData] =
 		useState<GetProjectDropdownOptionsQuery>()
+	const isValidProjectId = Number.isInteger(Number(projectId))
 
 	const { data: projectDropdownData } = useGetProjectDropdownOptionsQuery({
 		variables: { project_id: projectId! },
-		skip: !isLoggedIn || !projectId,
+		skip: !isLoggedIn || !isValidProjectId,
```

Comment:
Fixes an issue where we would try to query on pages like `/about_you` where we didn't have a valid project ID.